src/tests/akamai-cache-tag.ts

```diff
@@ -0,0 +1,7 @@
+import { GetCacheTag } from "../";
```

Comment:
This should be imported differently based on if we're testing the compiled library of the non-compiled one.

Reply:
I could use a suggestion on how to do this.

Reply:
I have removed the cache tag function from the compiled tests as it was unnecessary. I will explain this in person.